libs/langchain-community/src/utils/arxiv.ts

```diff
@@ -0,0 +1,211 @@
+import axios from 'axios'; // For HTTP requests
```

Comment:
Let's just use `fetch`

Reply:
Hi, sorry for the late response, but our team have just modified the file to use ```fetch``` instead of ```axios```. We have tested our code and the test cases still passed. Thank you!